partner_ref_unique/hooks.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Camptocamp SA
```

Comment:
Oh, we forget this! This can be fully removed and the hook removed in the manifest.

Reply:
Yes, thank you. Changes already made and pushed.